torch/nn/modules/linear.py

```diff
@@ -109,8 +109,10 @@ def extra_repr(self) -> str:
 class _LinearWithBias(Linear):
     bias: Tensor  # type: ignore[assignment]
 
-    def __init__(self, in_features: int, out_features: int) -> None:
-        super().__init__(in_features, out_features, bias=True)
+    def __init__(self, in_features: int, out_features: int, bias: bool = True,
+                 device=None, dtype=None) -> None:
+        super().__init__(in_features, out_features, bias=bias,
+                         device=device, dtype=dtype)
```

Comment:
I think adding `bias` as an argument defies the whole name of the module

Reply:
@z-a-f not sure what you mean by this, can you elaborate?

Reply:
I think he is saying it's contradictory that the module is called `_LinearWithBias` and yet it may or may not have a bias, based on the value of the `bias` flag.

Reply:
ah got it, thanks 😁

Reply:
...though I think the module does in fact need to pass the full set of Linear ctor params through for the fix to work, I think (given the call site in MultiheadAttention.__init__)

Reply:
You're correct about that, since we want to maintain the regression fix by having `bias` apply to both in / out projection layers.